Tests/Fuzzers/FuzzPacketParse.cpp

```diff
@@ -0,0 +1,24 @@
+#include <iostream>
+#include <IPv4Layer.h>
+#include <IPv6Layer.h>
+#include <unistd.h>
+#include <Packet.h>
+#include <PcapFileDevice.h>
+#include <RawPacket.h>
+
+extern "C" int LLVMFuzzerTestOneInput(const uint8_t *Data, size_t Size)
+{
+    struct timespec ts;
+    timespec_get(&ts, TIME_UTC);
+
+	if (Size < 2)
+		return 0;
+
```

Comment:
I think we want to suppress logs like we do today:
https://github.com/seladb/PcapPlusPlus/blob/1d29c00f441dc50650b500bb6b6fe72efce28990/Tests/Fuzzers/FuzzTarget.cpp#L47

To prevent redundant prints

Reply:
Noted! will fix